imcache_test.go

```diff
@@ -941,6 +942,53 @@ func TestImcache_GetAll_SlidingExpiration(t *testing.T) {
 	}
 }
 
+//GetAndRemove Tests Init
+func TestImcache_GetAndRemove(t *testing.T) {
+	for _, cache := range caches {
+		t.Run(cache.name, func(t *testing.T) {
+			c := cache.create()
+			c.Set("foo", "foo", WithNoExpiration())
+			c.Set("foobar", "foobar", WithExpiration(time.Hour))
+
+			//get foobar item and delete it
+			c.GetAndRemove("foobar")
+
+			got := c.GetAll()
+			want := map[string]string{
+				"foo": "foo",
+			}
+			if !reflect.DeepEqual(got, want) {
+				t.Errorf("imcache.GetAndRemove() = %v, want %v", got, want)
```

Comment:
Let's be consistent with the assertions in other tests, e.g. https://github.com/erni27/imcache/blob/master/imcache_test.go#L119 (the msg should match input param(s) and return value(s), if they are not really relevant, mark them as `_`.